@types/commercetools-frontend__ui-kit/index.d.ts

```diff
@@ -1,16 +1,1114 @@
 declare module '@commercetools-frontend/ui-kit' {
-  type CustomProperties = { [key: string]: string };
+  import * as React from 'react';
+  import { MessageDescriptor } from 'react-intl';
 
-  export const customProperties: CustomProperties;
+  export const version: string;
+
+  // Design tokens - customProperties
+  export {
+    default as customProperties,
+  } from '@commercetools-frontend/ui-kit/materials/custom-properties';
```

Comment:
This will be available when https://github.com/commercetools/ui-kit/pull/978 is merged and released.

For now, it's duplicated at the end of this file.